### PR TITLE
89 bug replies not showing

### LINKED ISSUE #89: BUG: Replies not showing
As a user, I would expect to see a reply that was posted.

**Current Behavior:**
- On submit of a reply, we see the console log that the reply has been sent, but we are not seeing the reply come back through content watcher and be displayed in the feed.

**Expected Behavior:**
- About a minute after submission, we should see the post populate with the reply.

Dev Note:
- Do not worry about the lack of a loading indicator. In another ticket.

### PR DESCRIPTION
**Pull request by claireolmstead**
# Purpose
The goal of this PR is to create the infrastructure to support replies.

Closes #89 

## Solution
The solution is to create a lightweight database that holds the replies, add styling for replies, and support on the frontend to display the replies.

@JoeCap08055 

### Change summary
* created sqllite db
* edited styling for replies
* add support for replies on backend and frontend

## Steps to Verify
1. Start everything from scratch
2. Login
3. Create a post
4. write a comment on your own post
5. write a comment on someone else's post
6. write a second comment.

## Additional details / screenshot
- *Not featured in this story:* the UX/UI case for if there are many replies.
- Screenshot 2024-07-25 at 5 14 10 PM

### CODE REVIEW

File: backend/db/contentDb.sqlite
Comment (JoeCap08055) [inline backend/db/contentDb.sqlite:1]: Remove; this file should not be committed

File: backend/services/AccountService.ts [hunk 1]
```diff
@@ -79,15 +78,15 @@ export class AccountService {
...
+  public async getAccount(msaId: string): Promise<Partial<AuthAccountResponse>> {
     try {
       const response = await this.client.AccountsControllerV1_getAccount(msaId);
       logger.debug(
         `AccountService: getAccount: Got account for msaID:(${msaId}), data:(${JSON.stringify(response.data)})`
       );
       return {
         msaId: msaId,
-        handle: response.data.handle,
+        handle: response.data.handle as { base_handle: string; canonical_base: string; suffix: number } | undefined,
```
Comment (JoeCap08055) [inline backend/services/AccountService.ts:89]: Note: created issue to have this handle type defined in the OpenAPI spec from account-service
https://github.com/AmplicaLabs/gateway/issues/303

File: backend/types/api.d.ts [hunk 2]
```diff
@@ -89,7 +91,11 @@ declare namespace Components {
     }
     export interface HandlesResponse {
       publicKey: string;
-      handle: string;
+      handle: {
+        base_handle: string;
+        canonical_base: string;
+        suffix: number;
+      };
```
Comment (JoeCap08055) [inline backend/types/api.d.ts:98]: Just confirming whether these handle-type changes are wanted; I noticed earlier that some code was out of sync with the OpenAPI spec, and things broke when I generated all the types. Perhaps a review of this warranted to make sure the spec & code are in sync & are what is desired.

Comment (claireolmstead) [reply backend/types/api.d.ts:98]: yes this all looks good!

**Review by JoeCap08055 — APPROVED** [review]

**Pull request merged**